### DISCUSSION TITLE
Automatic refetching without react hooks

### DISCUSSION THREAD

**Discussion by donald-richardson-dxd** [reduxjs/redux-toolkit#3199] [General]
I've set up my api slice via code splitting. Setting endpoints via injectEndpoints with the appropriate providesTags and invalidatesTags. I enhanceEndpoints to set addTagTypes as I add new types. So, I've set up automatic refetching. 

I'm running a mutation without react hooks, i.e. with `initiate`. My queries are not being refetched. Is this the expected behavior? 

If not then what might be going wrong?
If it is then how do I take advantage of automatic refreshing without using react hooks?

**Comment by markerikson** [comment]
@donald-richardson-dxd : that _should_ kick off the refetch.  Do you have the middleware added to the store?

**Reply by donald-richardson-dxd** [reply]
I'm injecting middleware dynamically using `redux-dynamic-middlewares` including the middleware from the slice. I've verified that the dynamic middleware approach works since other middleware that I'm injecting work fine.

**Reply by markerikson** [reply]
Can you post a CodeSandbox or record a Replay ( https://replay.io/record-bugs ) that shows this happening?

**Reply by donald-richardson-dxd** [reply]
False alarm. I misformatted the invalidatesTags. I accidentally put each element in an array: [[{type: "MyType": id: "myId"}], ["MyType"]].
